Batteries/Data/Array/Basic.lean

```diff
@@ -4,7 +4,6 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Arthur Paulino, Floris van Doorn, Jannis Limperg
 -/
 import Batteries.Data.List.Init.Attach
-import Batteries.Data.Array.Init.Lemmas
```

Comment:
This one could stay. The import is currently empty but its purpose is to be included here (and ideally nowhere else).

Reply:
Sure... importing empty files for which there are no active plans to add anything seems a bit of a waste of time, but no objection.